tests/lib/rules/no-unused-vars.js

```diff
@@ -141,11 +141,6 @@ ruleTester.run("no-unused-vars", rule, {
             output: null,
             errors: [{ type: "Identifier" }]
         },
-        {
-            code: "var a = b = c()",
-            output: null,
-            errors: [{ type: "Identifier" }]
-        },
```

Comment:
is it intentional to remove the case?

Reply:
Yes, since it's unsafe:

```js
var a = b = c

fn(b)
```

Reply:
IHMO, `var a = b = c();` => `b = c();`.

Reply:
It can't.

```js
let b
let a = b = c

fn(b)
```

It will be fixed as below, which is invalid:

```js
let b
let b = c

fn(b)
```

Reply:
```js
let a = b = c; //  => let a = (b = c);
```
so, the statement is to declare `a`, not `b` & `c`

Reply:
I know, but that auto fixing will produce error.

Reply:
I'm fine to not fix this case, but I don't think we should remove the case--just to prevent crashing